gui/src/main/java/org/verapdf/cli/multithread/reports/writer/MrrReportWriter.java

```diff
@@ -0,0 +1,64 @@
+package org.verapdf.cli.multithread.reports.writer;
```

Comment:
Again if `BaseCliRunner.ResultStructure` moves to https://github.com/veraPDF/veraPDF-library then so can this.